examples/all-clusters-app/all-clusters-common/include/camera-av-settings-user-level-management-instance.h

```diff
@@ -35,7 +35,7 @@ class AVSettingsUserLevelManagementDelegate : public Delegate
     bool CanChangeMPTZ() override;
 
     CHIP_ERROR LoadMPTZPresets(std::vector<MPTZPresetHelper> & mptzPresetHelpers) override;
-    CHIP_ERROR LoadDPTZStreams(std::vector<Structs::DPTZStruct::Type> dptzStream) override;
+    CHIP_ERROR LoadDPTZStreams(std::vector<Structs::DPTZStruct::Type> & dptzStreams) override;
```

Comment:
Please fix the commit message.  This is about basic correctness, not "consistency".